decidim-core/lib/decidim/amendable.rb

```diff
@@ -16,8 +16,7 @@ module Amendable
       has_many :emendations,
                through: :amendments,
                source: :emendation,
-               source_type: name,
-               inverse_of: :emendations
```

Comment:
Without this change we had `ActiveRecord::InverseOfAssociationRecursiveError` exceptions